widgets/urls.py

```diff
@@ -0,0 +1,13 @@
+"""
+WidgetApp urls
+"""
+from django.conf.urls import url
+from django.urls import include
+from rest_framework import routers
+
+from widgets.views import WidgetListViewSet
+
+router = routers.DefaultRouter()
+router.register(r"widget-lists", WidgetListViewSet, basename="widget_list")
```

Comment:
Can we change to `widget_lists` here, since we use underscores elsewhere in our REST API?